streams/src/test/java/org/apache/kafka/test/NoOpInternalValueTransformer.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.kafka.test;
+
+import org.apache.kafka.streams.kstream.ValueTransformerWithKey;
+import org.apache.kafka.streams.kstream.ValueTransformerWithKeySupplier;
+import org.apache.kafka.streams.processor.ProcessorContext;
+
+public class NoOpInternalValueTransformer<K, V> implements ValueTransformerWithKeySupplier<K, V, V> {
+    public ProcessorContext context;
```

Comment:
I don't think that the context should be a member of the supplier, but rather of the transformer -- otherwise, multiple transformers share the same context ?

Reply:
I agree, but this is a simple test class and switching it to be a member of the transformer muddies the test cases. How strongly do you feel about this?

Reply:
I see... Maybe rename to `SingletonNoOpInternalValueTransformer` and only create one `ValueTransformerWithKey` instance? This makes it more clear why it's designed this way? From the context of the class itself it's unclear -- only makes sense if you check it's usage in the test. Would be good to be self-descriptive IMHO.

Reply:
Done - also, 'Internal' part dropped inline with change that dropped the `InternalValueTranformerWithKey` interface